bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/ThingStatusDetail.java

```diff
@@ -23,7 +23,8 @@
     CONFIGURATION_ERROR(7),
     BRIDGE_OFFLINE(8),
     FIRMWARE_UPDATING(9),
-    DUTY_CYCLE(10);
+    FIRMWARE_UPDATE_PENDING(10),
```

Comment:
There are Zigbee devices which run with batteries and have a energy saving standby mode. If such a device is in standby mode it does not start the update -> FIRMWARE_UPDATE_PENDING. If it becomes active again it starts the pending update automatically  -> FIRMWARE_UPDATING.

So the semantics are "There is a queued update which was not started yet"